replication.go

```diff
@@ -0,0 +1,201 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package simplex
+
+import (
+	"bytes"
+
+	"go.uber.org/zap"
+)
+
+type Request struct {
+	FinalizationCertificateRequest *FinalizationCertificateRequest
+	LatestRoundRequest             *LatestRoundRequest
+}
+
+type Response struct {
+	FinalizationCertificateResponse *FinalizationCertificateResponse
+	LatestRoundResponse             *LatestRoundResponse
+}
+
+// request a finalization certificate for the given sequence number
+type FinalizationCertificateRequest struct {
```

Comment:
I think this should be:

```
type FinalizationCertificateRequest struct {
	Sequences []uint64
}
```

Because what if we want to ask from a single node, 100 sequences? Will we send it 100 different messages? If the remote node has a buffer that restricts message number, it will overflown.